packages/mds-agency/request-handlers.ts

```diff
@@ -253,19 +277,19 @@ export const submitVehicleEvent = async (req: AgencyApiRequest, res: AgencyApiRe
     const message = err.message || String(err)
     if (message.includes('duplicate')) {
       logger.info(name, 'duplicate event', event.event_type)
-      res.status(409).send({
-        error: 'duplicate_event',
-        error_description: 'an event with this device_id and timestamp has already been received'
+      res.status(400).send({
+        error: 'bad_param',
+        error_description: 'An event with this device_id and timestamp has already been received'
       })
     } else if (message.includes('not found') || message.includes('unregistered')) {
       logger.info(name, 'event for unregistered', event.device_id, event.event_type)
       res.status(400).send({
-        error: 'unregistered',
-        error_description: 'the specified device_id has not been registered'
+        error: 'bad_param',
```

Comment:
Should be `unregistered` per https://github.com/openmobilityfoundation/mobility-data-specification/tree/dev/agency#vehicle---event